docs/UserGuide.md

```diff
@@ -3,24 +3,40 @@ layout: page
 title: User Guide to TR4CKER
 ---
 
+1. [Overview](#Overview)
+    1. [About TR4CKER](#1.1.About TR4CKER)
+    2. [About this User Guide](#1.2.About this User Guide)
+2. [Quick Start](#2.Quick Start)    
+3. [Commands]()
+
+## 1. Overview
+
+### 1.1. About TR4CKER     
 TR4CKER is a desktop app that helps SoC students track their tasks, meet their deadlines, and stay productive. It is 
 optimized for Computing students familiar with Command Line Interface (CLI), who can manage their tasks efficiently by typing in commands.
 
-* Table of Contents
-{:toc}
+### 1.2. About this User Guide
+This User Guide explains how to use TR4CKER, as well as provides an understanding of the features and commands and some 
+common use cases of this application.  
+  
+In this guide, we cover:
+1. How to navigate the Graphical User Interface (GUI) 
+2. How to use the Command Line Interface (CLI)
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## 2. Quick Start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-2. Download the latest `tr4cker.jar` from [here](https://github.com/AY2021S1-CS2103T-T10-2/tp/releases/tag/v1.1).
+2. Download the latest `tr4cker.jar` from [here](https://github.com/AY2021S1-CS2103T-T10-2/tp/releases/tag/v1.3).
```

Comment:
Maybe can link straight to just tp/releases for the latest release instead of specifically /tag/v1.3